### ISSUE TITLE
"airflow upgradedb" succeeds but "airflow db upgrade" fails on 1.10.14

### ISSUE THREAD

**Issue by jsmodic** [apache/airflow#13205]
**Apache Airflow version**:
1.10.14

**What happened**:

> airflow upgradedb
[2020-12-21 00:47:14,079] {db.py:378} INFO - Creating tables
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
>
> echo $?
0

> airflow db upgrade
(no output)
>
> echo $?
1

**What you expected to happen**:

"airflow db upgrade" to exit 0, or at least output something useful.

**How to reproduce it**:

I updated from 1.10.12 to 1.10.14.

**Anything else we need to know**:

**Comment by potiuk** [comment]
I think you are somehow redirecting stderr and do not see what is being written there. This is the message printed to stderr when you run `arflow db upgrade`

```
airflow db upgrade

Please install apache-airflow-upgrade-check distribution from PyPI to perform upgrade checks

root@88e40ae01f74:/opt/airflow# airflow db upgrade 2>/dev/null
root@88e40ae01f74:/opt/airflow# 
```

**Comment by jsmodic** [comment]
That must be my misunderstanding, I cut out that output as I thought it was part of the the deprecation warnings (that I have a ton of plugin-related ones).

I was under the impression the two commands were aliases in 1.10.14, so I was migrating my tooling to begin support for 2.0. I guess that's just a battle to fight in 2.0 instead.

**Comment by potiuk** [comment]
I believe they are not. They were added very recently (@ashb ? ) and my current thinking was that they were there there to raise awareness about 2.0 migration rather than add the features from 2.0 to 1.10.

**Comment by ashb** [comment]
Oh that looks like a bug in `db upgrade` -- it's bound to the wrong function maybe.

**Comment by ashb** [comment]
@potiuk I added the "new style" commands to 1.10.14 so that it would be possible to upgrade any scripts etc to use new style commands. So this is a bug

**Comment by jsmodic** [comment]
https://github.com/apache/airflow/blob/1.10.14/airflow/bin/cli.py#L3114

looks like an easy fix there

**Comment by potiuk** [comment]
> https://github.com/apache/airflow/blob/1.10.14/airflow/bin/cli.py#L3114
> 
> looks like an easy fix there

Yeah. Can you please make a PR fixing it ?

**Comment by jsmodic** [comment]
Sure, that's just a one line change: https://github.com/apache/airflow/pull/13267

**Comment by potiuk** [comment]
Fixed by #13267 :)
